src/cli.js

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env node
+const { ensureFile, writeJson } = require("fs-extra");
+const path = require("path");
+const program = require("commander");
+const axeRunner = require("./axe-runner");
+
+const runTestsInPage = require("./run-tests-in-page");
+
+program
+  .requiredOption(
+    "-t, --testsJson <testsJson>",
+    "Path to JSON file containing all ACT Rules testcases"
+  )
+  .requiredOption(
+    "-d, --testsDir <testsDir>",
+    "Directory containing ACR testcases assets and files"
+  )
+  .requiredOption("-s, --siteUrl <siteUrl>", "Url of the ACT Rules site")
+  .option(
+    "-o, --outfile <jsonFile>",
```

Comment:
I see how this being used to fetch any cached results, if file already exists.

Is the intention to save the previous results as an artifact in CI, so this can be passed in as an argument?

Reply:
We might use it that way, sure. I built this because I couldn't stand a build script that took 30 minutes to run every time I used it.